dev-requirements.txt

```diff
@@ -9,6 +9,7 @@ cryptography
 html5lib  # needed for beautifulsoup
 jupyterlab >=3
 mock
+nbclassic  # ensures /tree/ handler is present as assumed by tests
```

Comment:
nbclassic is already a transitive dependency via jupyterlab, but let's be explicit since we call it directly and jupyterlab will probably drop the dependency someday